src/domain/session/room/RoomViewModel.js

```diff
@@ -197,18 +197,80 @@ export class RoomViewModel extends ViewModel {
         }
     }
     
+    async _getMessageInformations (message) {
+        let msgtype = "m.text";
+        if (message.startsWith("/")) {
+            let command=message.substring(1).split(" ");
+            switch (command[0]) {
+                case "me":
+                   message = message.substring(4).trim();
+                   msgtype = "m.emote";
+                   break;
+               case "join":
+                   if (command.length==2) {
+                       let roomname = message.substring(5).trim();
+                       try {
+                           await this._options.client.session.joinRoom(roomname);
+                       } catch (exc) {
+                           if (exc.statusCode??exc.status==400) {
```

Comment:
We always add a space before and after operators:
```suggestion
                           if (exc.statusCode ?? exc.status === 400) {
```
Can you make this change throughout? It should be possible to setup your IDE/editor to do this automatically.